scripts/lint

```diff
@@ -1,14 +1,14 @@
 #!/bin/sh -e
 
-export PREFIX=""
+PACKAGE="httpcore"
+
 if [ -d 'venv' ] ; then
-    export PREFIX="venv/bin/"
+    PREFIX="venv/bin/"
+else
+    PREFIX=""
 fi
-export SOURCE_FILES="httpcore tests"
 
 set -x
 
-${PREFIX}autoflake --in-place --recursive $SOURCE_FILES
-${PREFIX}isort --project=httpcore --recursive --apply $SOURCE_FILES
```

Comment:
`isort` is removed from here but not added to `check`, should it be there or are we dropping it?

Reply:
I'm not sure actually. There's a little niggle with it at the moment, in that `isort` seems to be making one adjustment that black then reverts, which means we get noisy outputs even when everything is clean.

Reply:
I've noticed that too but seems we end up with a pretty consistent setup, albeit a noisy one.

I'm not too fussed either way, but if we do decide to drop `isort` we probably want to remove it from the requirements as well.